packages/client/wallets/aa/src/CrossmintAASDK.ts

```diff
@@ -1,92 +1,91 @@
-import type { SmartAccountSigner } from "@alchemy/aa-core";
-import { ZeroDevEthersProvider, convertEthersSignerToAccountSigner } from "@zerodev/sdk";
-import { Signer } from "ethers";
+import type { SmartAccountSigner } from '@alchemy/aa-core';
```

Comment:
i think your editor must be using a different prettier config, we use 4 spaces and double quotes